develop/dev-guide-create-table.md

```diff
@@ -362,53 +358,52 @@ Running results:
+- Use meaningful column names. It is recommended that you follow your company or organization's table naming convention. If your company or organization does not have a corresponding naming convention, refer to the [column naming convention](/develop/dev-guide-object-naming-guidelines.md#field-naming-convention).
 
-### Rules to follow when selecting primary key
+### Guidelines to follow when selecting primary key
 
 - Define a **primary key** or **unique index** within the table.
 - Try to select meaningful **columns** as **primary keys**.
-- For performance reasons, try to avoid storing extra-wide tables, the number of table fields is not recommended to over `60`, the total data size of a single row is not recommended to over `64K`, and fields with too much data length are best split to another table.
+- For performance reasons, try to avoid storing extra-wide tables. It is not recommended that the number of table fields is over `60` and that the total data size of a single row is over `64K`. It is recommended to split fields with too much data length to another table.
 - It is not recommended to use complex data types.
-- The fields need to **JOIN** , ensure data types absolute consistency, to avoid implicit conversion.
-- Avoid defining **primary keys** on a single monotonic data column. If you use a single monotonic data column (e.g., a column of `AUTO_INCREMENT`) to define the **primary key**, there is a risk of a negative impact on write performance. If possible, use `AUTO_RANDOM` instead of `AUTO_INCREMENT` (this loses the continuous and incremental nature of the primary key).
-- If you **_must_** create an index on a single monotonic data column at write-intensive scenarios. Instead of defining this monotonic data column as the **primary key**. You can use `AUTO_RANDOM` to create the **primary key** for that table, or use [SHARD_ROW_ID_BITS](/shard-row-id-bits.md) and [PRE_SPLIT_REGIONS](/common/sql-statements/sql-statement-split-region.md#pre_split_regions) to shard `_tidb_rowid`.
+- For the fields to be joined, ensure that the data types are consistent and avoid implicit conversion.
+- Avoid defining **primary keys** on a single monotonic data column. If you use a single monotonic data column (for example, a column with the `AUTO_INCREMENT` attribute) to define the **primary key**, it might impact the write performance. If possible, use `AUTO_RANDOM` instead of `AUTO_INCREMENT`, which discards the continuous and incremental attribute of the primary key.
+- If you really need to create an index on a single monotonic data column at write-intensive scenarios, instead of defining this monotonic data column as the **primary key**, you can use `AUTO_RANDOM` to create the **primary key** for that table, or use [`SHARD_ROW_ID_BITS`](/shard-row-id-bits.md) and [`PRE_SPLIT_REGIONS`](/common/sql-statements/sql-statement-split-region.md#pre_split_regions) to shard `_tidb_rowid`.
```

Comment:
```suggestion
- If you really need to create an index on a single monotonic data column at write-intensive scenarios, instead of defining this monotonic data column as the **primary key**, you can use `AUTO_RANDOM` to create the **primary key** for that table, or use [`SHARD_ROW_ID_BITS`](/shard-row-id-bits.md) and [`PRE_SPLIT_REGIONS`](/sql-statements/sql-statement-split-region.md#pre_split_regions) to shard `_tidb_rowid`.
```